lib/mayaUsd/ufe/UsdAttribute.h

```diff
@@ -29,10 +29,18 @@
 #endif
 
 // Ufe::Attribute overrides (minus the type method)
+#ifdef UFE_HAS_DISPLAY_NAME
+#define UFE_V5_ATTRIBUTE_OVERRIDES \
+    std::string displayName() const override { return UsdAttribute::_displayName(); }
+#else
+#define UFE_V5_ATTRIBUTE_OVERRIDES
+#endif
+
 #ifdef UFE_V4_FEATURES_AVAILABLE
 #define UFE_ATTRIBUTE_OVERRIDES                                                               \
     bool        hasValue() const override { return UsdAttribute::_hasValue(); }               \
     std::string name() const override { return UsdAttribute::_name(); }                       \
+    UFE_V5_ATTRIBUTE_OVERRIDES                                                                \
```

Comment:
This and the lines 32-37 could just be three lines, given that this isn't used anywhere else.

```
#ifdef UFE_HAS_DISPLAY_NAME
    std::string displayName() const override { return UsdAttribute::_displayName(); }
#endif
```

Reply:
NO, I need to declare a macro because I don;t think you can have a #ifdef in the middle of a #define.

Reply:
Honestly, I think this whole slew of ifdef made by LookdevX is terrible. I wish they'd done ifdef on THEIR codebase, by declaring a façade class or something instead of ugliying our code.

Reply:
I see. I missed the define. Yes it's impossible to ifdef inside a define.